cmd/kubectl/kubectl.go

```diff
@@ -35,6 +35,7 @@ import (
 func main() {
 	rand.Seed(time.Now().UnixNano())
 
+	// 创建 kubectl 命令
```

Comment:
I am not sure if adding Chinese comments are a good idea because if other people come in and add other languages it will be cumbersome.

Reply:
sorry, I upload my comments to the wrong branch, I will fix it right now.